Base: Remove or replace test files with questionable licensing state

BenWiederhake opened the pull request:
This PR:
- Removes bmpsuite as discussed in #3399
- Removes an unlicensed copy of BMFW, which we used as a "test page" for the browser. However, virtually *all* pages on the internet test more functionality than BMFW, by construction. Even [this one](https://benwiederhake.github.io/freenode-serenity-archive/quote-100995.html), simple as it is.

This leaves only `AK/Checked.h` and `AK/Function.h` as far as I can see. As these are basically unavoidable and have changed sufficiently over time, I'll say:

Closes #3399.

Note that I also checked all other files in `Base/res/html/`, and most files seem like we just made them ourselves, or are simple enough to not warrant any further inspection, with a few notable exceptions:
- `more-attribute-selectors.html` was apparently added by the author Daniel Glazman himself in fdf1c3c2f1bf8ffcadb9c5515da25451520d7b4a. Therefore it should be implicitly covered by the Serenity license, I hope.
- `car.png` is probably a photo by @MacDue, committed in f807fe6f6ced06036e9e8f6707ce725d36d8830c – can you confirm that you made the photo? Google and [TinEye](https://tineye.com/search/f47bc487100343bbae008f96e9e9bc6a8813b44d) haven't seen this image before (Why not even on GitHub?)
- `old-computer.png` looks too old to be a photo, committed by @MacDue in 51899751d77f51b744aec6179a60dd9138d74bb4 – can you confirm that you have the permission from the person in the photo and the permission of the photographer to share it with the world under the BSD license? Google and [TinEye](https://tineye.com/search/5536e9532ece9cd96946616fa497fb1a6633286a) haven't seen this image before (Why not even on GitHub?)

MacDue commented:
I can confirm I own both car.png and old-computer.png, and can license them as BSD-3-Clause.

BenWiederhake commented:
Changes since last push:
- Rebased to clear CI failure

![Bildschirmfoto_2023-05-26_23-09-09](https://github.com/SerenityOS/serenity/assets/2690845/2cef9348-9821-4470-9a18-62e91941b871)

I can't reproduce that error, so let's just hope it's just CI flakes.

skyrising commented:
> This leaves only AK/Checked.h and AK/Function.h as far as I can see

The imported files I could find, by license:

- BSD 2-Clause
  - `AK/Checked.h`: Apple Inc.
  - `AK/Function.h`: Apple Inc.
  - `Userland/Libraries/LibJS/SafeFunction.h`: Apple Inc.
- BSD 3-Clause
  - `Userland/Libraries/LibC/elf.h`: Erik Theisen/OpenBSD
- LGPLv2
  - `Kernel/FileSystem/Ext2FS/Definitions.h`: Remy Card & Linus Torvalds from `e2fsprogs/lib/ext2fs/ext2_fs.h`

GMTA commented on Base/res/html/misc/welcome.html:
```diff
@@ -211,7 +211,7 @@ <h2>Image Formats</h2>
             <li><a href="pngsuite_siz_png.html">pngsuite odd sizes test</a></li>
```
Commit typo: Ladybirdy

BenWiederhake replied:
That's just compensation for [`LADBIRD`](https://github.com/SerenityOS/serenity/pull/18682#issue-1698677221) xD

GMTA reviewed (approved):
Just a tiny typo!

BenWiederhake commented:
Changes since last push:
- Fixed a silly typo (Thanks @GMTA!)
- Rebased for convenience

BenWiederhake commented:
CI failure seems to be due to broken master. :(

BenWiederhake commented:
Changes since last push:
- Rebased to clear CI flake, and also convenience
